src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+# Positron Emission Tomography
+
+Support for Positron Emission Tomography (PET) was developed as a [BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please cite the following paper when referring to this part of the standard in
+context of the academic literature:
+
+> Knudsen GM, Ganz M, Appelhoff S, Boellaard R, Bormans G, Carson RE, Catana C,
+> Doudet D, Gee AD, Greve DN, Gunn RN, Halldin C, Herscovitch P, Huang H, Keller SH,
+> Lammertsma AA, Lanzenberger R, Liow JS, Lohith TG, Lubberink M, Lyoo CH, Mann JJ,
+> Matheson GJ, Nichols TE, Nørgaard M, Ogden T, Parsey R, Pike VW, Price J, Rizzo G,
+> Rosa-Neto P, Schain M, Scott PJH, Searle G, Slifstein M, Suhara T, Talbot PS,
+> Thomas A, Veronese M, Wong DF, Yaqub M, Zanderigo F, Zoghbi S, Innis RB.
+> Guidelines for Content and Format of PET Brain Data in Publications
+> and in Archives: A Consensus Paper.
+> Journal of Cerebral Blood Flow and Metabolism, 2020 Aug;40(8):1576-1585.
+> doi: [10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433)
+
+The following example PET datasets have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+-   One subject single dynamic scan (pet, mri, blood):
+	[`pet_example1`](https://www.dropbox.com/sh/dmjmaxfswatptpm/AACw31cTyChLA1kwM4Vhqueja?dl=0)
+-   One subject single dynamic scan (pet, mri):
+	[`pet_example2`](https://www.dropbox.com/sh/i95a71rpq2unks7/AACby5XcYQp9vcwgx3cNXBhza?dl=0)
+-   One subject single dynamic scan (pet, blood):
+	[`pet_example3`](https://www.dropbox.com/sh/i0c3iq50id4s0d3/AABSjaczHM1SQnMRvh3fc7fna?dl=0)
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## PET recording data
+
+As specified above, this extension is relevant for brain PET imaging
+and its associated data, such as blood data.
+In addition, the extension is in accordance with the guidelines for reporting
+and sharing brain PET data (Knudsen et al. 2020, [1]).
+
+*Shared MRI data along with PET:*
+To share structural magnetic resonance (MR) images with your PET data,
+please follow the original BIDS specification
+(https://bids-specification.readthedocs.io/en/stable).
+Please pay specific attention to the format the MR images are in,
+such as if they have been unwarped in order to correct for gradient non-linearities.
+There is a specific field in the MRI BIDS (https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) called “NonlinearGradientCorrection”
+which indicates this.
```

Comment:
```suggestion
Please pay specific attention to the format the MR images are in.
For example, if they have been unwarped to correct for gradient non-linearities, it should be indicated in a specific field of the MRI-BIDS .json file (https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) called “NonlinearGradientCorrection”.
```

Reply:
"Please pay specific attention to the format the MR images are in."
Is this really a "format"? I'm not big expert in MRI but sounds like this word doesn't fit well

Reply:
Hi @effigies @Remi-Gau @tsalo @sappelhoff and @mnoergaard, can we discuss this tomorrow?